docssrc/source/tutorials/custom_cleaner/custom.json

```diff
@@ -14,12 +14,13 @@
     "features": {
         "excerpt": {
             "encoder": {
-                "module": "Rich_Text.PretrainedLangEncoder",
+                "module": "PretrainedLangEncoder",
                 "args": {
                     "output_type": "False",
```

Comment:
how can output type be false?

we should standardize data types going in

Reply:
This is a mistake, types are standardized, but not enforced because enforcign this in python has to do with 3rd party tools

Reply:
Also that is and incorrect file that's not used anywhere, this shluld be deleted, ups xD